package.json

```diff
@@ -35,8 +35,23 @@
     "@babel/eslint-parser": "^7.17.0",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
+    "@types/cli-table": "^0.3.2",
+    "@types/fs-extra": "^11.0.3",
+    "@types/libnpmsearch": "^2.0.4",
+    "@types/lodash": "^4.14.199",
+    "@types/mocha": "^10.0.3",
+    "@types/node": "^14.18.63",
+    "@types/npmlog": "^4.1.4",
+    "@types/pkginfo": "^0.4.1",
+    "@types/promptly": "^3.0.3",
+    "@types/request": "^2.48.11",
+    "@types/should": "^13.0.0",
+    "@types/test-console": "^2.0.1",
+    "@types/update-notifier": "^6.0.5",
+    "@typescript-eslint/eslint-plugin": "^6.8.0",
```

Comment:
I received a few warnings for `EBADENGINE` when running `npm install`. All of them are related to `@typescript-eslint` or its dependencies, such as `ts-api-utils`. Initially, I thought it was related to ESLint, which is a dev dependency and should not affect users who install the package. However, even running `npm install --omit=dev` resulted in the same error.

```plaintext
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: '@typescript-eslint/eslint-plugin@6.8.0',
npm WARN EBADENGINE   required: { node: '^16.0.0 || >=18.0.0' },
npm WARN EBADENGINE   current: { node: 'v14.19.2', npm: '9.6.7' }
npm WARN EBADENGINE }
...

npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'ts-api-utils@1.0.3',
npm WARN EBADENGINE   required: { node: '>=16.13.0' },
npm WARN EBADENGINE   current: { node: 'v14.19.2', npm: '9.6.7' }
npm WARN EBADENGINE }
```

It appears that the development environment requires Node.js v16. This implies that we should develop and test everything in Node.js v16. To address this, we might consider requiring an upgrade to Node.js v16, which can be done by updating the .nvmrc file and the `engines` field in the package.json.

Reply:
Please also check if the README mentions the need for Node v16.

Reply:
Alright will do

Reply:
See d53b70b9b7791cda87b4b8c9260e23176a59fbce

Reply:
https://github.com/openupm/openupm-cli/commit/d53b70b9b7791cda87b4b8c9260e23176a59fbce LGTM